vignettes/render-with-ci.qmd

```diff
@@ -0,0 +1,95 @@
+---
+title: "Render with CI"
+description: >
+  How to render with CI
+vignette: >
+  %\VignetteIndexEntry{Render with CI}
+  %\VignetteEngine{quarto::html}
+  %\VignetteEncoding{UTF-8}
+---
+
+If you want to render and publish your multilingual book or website with CI, you must render your project with `render_book()` or `render_website()`, respectively. Don't use Quarto's own render and publish steps.
+
+## Github Actions and Github Pages
+
+If you want to use GitHub Pages to deploy your website, use the Quarto CLI to setup the branch for you:
+
+```bash
+quarto publish gh-pages
+```
+
+This will create a new branch called `gh-pages` in your repository. **Note: as your project was rendered using `quarto render` it won't be multilingual yet.**
+
+You can then setup your GitHub Actions. In your repository in `.github/workflows/` create a file named `publish.yml` with the following content.
```

Comment:
do we really want to maintain this workflow example? Could we instead explain what file to take as starting point then what steps to add? With a link to an example on GitHub from a "real" multilingual project.

Reply:
Maybe we need a list of things that need to happen

- install software including Quarto and R dependencies (stored in DESCRIPTION or renv)
- rendering using babelquarto
- pushing the HTML somewhere where it is deployed (a GitHub pages branch, Netlify etc)